### PR TITLE
Add Fireworks.app

### PR DESCRIPTION
**Pull request by byassine52**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/fireworks.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'fireworks' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.fireworksapp.xyz/public/fireworks.zip'
```
Comment (vitorgalvao) [inline Casks/fireworks.rb:5]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) (`https://www.fireworksapp.xyz`).

File: Casks/fireworks.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'fireworks' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/fireworks.rb:2]: Add a version and shasum.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/fireworks.rb (hunk 2 above)
Comment (adidalal) [reply Casks/fireworks.rb:2]:
```
version '1.2'
sha256 925774f32399c6b11e8e07cab5fb39f30c5bf7df6896540992065e5bac0327cd
```

File: Casks/fireworks.rb (hunk 1 above)
Comment (byassine52) [reply Casks/fireworks.rb:5]: Sorry for the delay.
Done 👍

File: Casks/fireworks.rb (hunk 2 above)
Comment (byassine52) [reply Casks/fireworks.rb]: Sorry for the delay.
Done 👍

**Comment by byassine52** [comment]
Applied requested changes.

File: Casks/fireworks.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
+
+  url 'https://www.fireworksapp.xyz/public/fireworks.zip'
+  name 'Fireworks'
+  homepage 'https://www.fireworksapp.xyz/'
+
+  app 'Fireworks.app'
+end
```
Comment (ran-dall) [inline Casks/fireworks.rb:11]:
```suggestion

  zap trash: [
             '~/Library/Application Scripts/com.Besher-Al-Maleh.fireworks',
             '~/Library/Containers/com.Besher-Al-Maleh.fireworks',
           ]
end
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
@byassine52 You're also missing the `zap` stanza.

File: Casks/fireworks.rb (hunk 3 above)
Comment (byassine52) [reply Casks/fireworks.rb:16]: Great addition @ran-dall 😄 👏

**Pull request merged**